justbuild.it-WebApp/src/main/resources/templates/add.html

```diff
@@ -1,60 +1,131 @@
-      <option value="2">Two</option>
-      <option value="3">Three</option>
-    </select>
-  </div>
-  <div class="input-group mb-3">
-    <label class="input-group-text" for="inputGroupSelect03">Słowo kluczowe</label>
-    <select class="form-select" id="inputGroupSelect03">
-      <option selected>Wybierz...</option>
-      <option value="1">One</option>
-      <option value="2">Two</option>
-      <option value="3">Three</option>
-    </select>
-  </div>
-  <div>
-    <button type="submit" class="btn btn-primary">Zatwierdź</button>
-  </div>
+<div th:insert="~{fragments/header :: header-add}"></div>
 
+<form id="addForm" class="addForm col-11 mt-5 mb-5 mx-auto" action="#" th:action="@{/add}" th:object="${offer}"
+      method="post">
+    <hr>
+    <div class="id mb-4 mt-3 col-1">
+        <label for="formGroupExampleInput0">ID oferty: </label>
+        <input type="text" class="form-control" id="formGroupExampleInput0" th:field="*{offerID}" readonly>
+    </div>
+    <hr/>
+    <div class="mb-4 mt-5">
+        <input type="text" class="form-control" id="formGroupExampleInput2" th:field="*{city}"
+               placeholder="Tu wpisz miasto*" required>
+        <div th:if="${#fields.hasErrors('city')}" th:errors="*{city}" class="error-message"></div>
+    </div>
+    <div class="mb-4">
+        <input type="text" class="form-control" id="formGroupExampleInput3" th:field="*{userFirstName}"
+               placeholder="Tu wpisz imię">
+    </div>
+    <div class="mb-4">
+        <input type="text" class="form-control" id="formGroupExampleInput4" th:field="*{userLastName}"
+               placeholder="Tu wpisz nazwisko">
+    </div>
+    <div class="mb-4">
+        <input type="text" class="form-control" id="formGroupExampleInput5" th:field="*{userCompanyName}"
+               placeholder="Tu wpisz nazwę firmy*" required>
+        <div th:if="${#fields.hasErrors('userCompanyName')}" th:errors="*{userCompanyName}" class="error-message"></div>
+    </div>
+    <div class="mb-4">
+        <input type="text" class="form-control" id="formGroupExampleInput6" th:field="*{userTelephoneNumber}"
+               placeholder="Tu wpisz telefon kontaktowy*" required>
+        <div th:if="${#fields.hasErrors('userTelephoneNumber')}" th:errors="*{userTelephoneNumber}"
+             class="error-message"></div>
+    </div>
+    <div class="mb-4">
+        <input type="email" class="form-control" id="formGroupExampleInput7" th:field="*{userEmailAddress}"
+               placeholder="Tu wpisz email*" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$">
+        <div th:if="${#fields.hasErrors('userEmailAddress')}" th:errors="*{userEmailAddress}"
+             class="error-message"></div>
+    </div>
+    <div class="mb-5">
+        <input type="text" class="form-control" id="formGroupExampleInput1" th:field="*{offerContent}"
+               placeholder="Tu wpisz treść ogłoszenia">
+    </div>
+    <div class="input-group mb-4">
```

Comment:
Tutaj pod formularzem z treścią ogłoszenia( tak jak sugerował Mariusz) dałbym informację, że pola oznaczone gwiazdką są wymagane. Albo całkiem na dole pod wszystkimi formularzami :P Ale to nie koniecznie na teraz.

Reply:
dodałem informację że pola są wymagane w placeholderach